source/client-side-encryption/tests/countDocuments.json

```diff
@@ -1,252 +1,257 @@
-                            "filter": {
-                                "$or": [
-                                    {
-                                        "_id": {
-                                            "$in": [
-                                                {
-                                                    "$binary": {
-                                                        "base64": "AAAAAAAAAAAAAAAAAAAAAA==",
-                                                        "subType": "04"
-                                                    }
-                                                }
-                                            ]
-                                        }
-                                    },
-                                    {
-                                        "keyAltNames": {
-                                            "$in": []
-                                        }
-                                    }
-                                ]
-                            },
-                            "$db": "admin"
-                        },
-                        "command_name": "find"
+                  "$match": {
+                    "encrypted_string": {
+                      "$eq": {
+                        "$binary": {
+                          "base64": "AQAAAAAAAAAAAAAAAAAAAAACwj+3zkv2VM+aTfk60RqhXq6a/77WlLwu/BxXFkL7EppGsju/m8f0x5kBDD3EZTtGALGXlym5jnpZAoSIkswHoA==",
+                          "subType": "06"
+                        }
+                      }
                     }
+                  }
                 },
                 {
-                    "command_started_event": {
-                        "command": {
-                            "aggregate": "default",
-                            "pipeline": [
-                                {
-                                    "$match": {
-                                        "encrypted_string": {
-                                            "$binary": {
-                                                "base64": "AQAAAAAAAAAAAAAAAAAAAAACwj+3zkv2VM+aTfk60RqhXq6a/77WlLwu/BxXFkL7EppGsju/m8f0x5kBDD3EZTtGALGXlym5jnpZAoSIkswHoA==",
-                                                "subType": "06"
-                                            }
-                                        }
-                                    }
-                                },
-                                {
-                                    "$group": {
-                                        "_id": 1,
-                                        "n": {
-                                            "$sum": 1
-                                        }
-                                    }
-                                }
-                            ]
-                        },
-                        "command_name": "aggregate"
+                  "$group": {
+                    "_id": {
+                      "$const": 1
```

Comment:
The real change 2: Also, `mongocryptd` adds `$const` node here and in several lines below.